util/stringutil/redact.go

```diff
@@ -0,0 +1,25 @@
+package stringutil
+
+import "net/url"
+
+func RedactCredentials(s string) string {
```

Comment:
I don't think we need new pkg for this function. I think it can even be a private function. If needs to be public then reuse `gitutil`

Reply:
sounds good, i'll make it private.